typescript/src/accounts/evm/toEvmServerAccount.ts

```diff
@@ -32,26 +34,29 @@ export function toEvmServerAccount(
       const result = await apiClient.signEvmMessage(options.account.address, {
         message: message.toString(),
       });
-      return result.signature as `0x${string}`;
+      return result.signature as Hex;
     },
 
     async sign(parameters: { hash: Hash }) {
       const result = await apiClient.signEvmHash(options.account.address, {
         hash: parameters.hash,
       });
-      return result.signature as `0x${string}`;
+      return result.signature as Hex;
     },
 
     async signTransaction(transaction: TransactionSerializable) {
       const result = await apiClient.signEvmTransaction(options.account.address, {
         transaction: serializeTransaction(transaction),
       });
-      return result.signedTransaction as `0x${string}`;
+      return result.signedTransaction as Hex;
     },
 
     async signTypedData() {
       throw new Error("Not implemented");
     },
+    async transfer(transferArgs): Promise<TransferResult> {
+      return transfer(apiClient, account, transferArgs);
```

Comment:
This is circular-ish (it's referencing `account`, which is itself being defined by this function. Will test to make sure it works in environments that are less tolerable to circular deps, otherwise will think about how to break this seemingly circular code path

Reply:
Validated this works in a fresh Next.js app without any custom config